services/booking/current_prices.rb

```diff
@@ -0,0 +1,28 @@
+class BookingService
+  class CurrentPrices < ApplicationService
+    def initialize(params)
+      @first_class_price  = params[:first_class_price].to_f
+      @second_class_price = params[:second_class_price].to_f
+      @price_coefficient  = params[:price_coefficient].to_f
+    end
+
+    def call
+      {
+        first_class:  first_class_current_price,
+        second_class: second_class_current_price
+      }
+    end
+
+    private
+
+    attr_reader :first_class_price, :second_class_price, :price_coefficient
+
+    def first_class_current_price
+      first_class_price * price_coefficient
```

Comment:
`first_class_current_price` & `second_class_current_price` contain the same logic with different argument. You can extract it to a generic method, e.g
```ruby
def booking_price(price)
  price * price_coefficient
end
```